### ISSUE TITLE
Negative Scores for cross-encoder/ms-marco-MiniLM-L-6-v2

### ISSUE THREAD

**Issue by zynos** [UKPLab/sentence-transformers#1058]
Hello everyone

when i run this line with the model "cross-encoder/ms-marco-MiniLM-L-6-v2"
`        cross_scores = self.cross_encoder.predict(cross_inp)
`
the score contains negtaive values, e.g -8.3. For other models it is in the desired 0 -1 range.

when I add the sigmoid activation function as an argument, the values also lie in the 0-1 range

`        cross_scores = self.cross_encoder.predict(cross_inp, activation_fct=nn.Sigmoid())
`

Is this the proper way to solve this problem and should sigmoid not be the default activation function anyways?

Thanks in advance!

**Comment by nreimers** [comment]
Hi,
this was changed in a recent version. The activation function is set as a parameter for the models. For the named model, the logits are returned due to how the model was trained (model was trained to learn the logits of an ensemble of large models).

For ranking it does not make a difference. You can apply Sigmoid if you like, but as mentioned, the ranking of results will stay the same.

**Comment by zynos** [comment]
Thank you for the explanation :)
